rust/src/delta.rs

```diff
@@ -887,6 +938,20 @@ pub async fn open_table_with_version(
     Ok(table)
 }
 
+/// TODO
+pub async fn open_table_with_version_and_log_buffer(
+    table_uri: impl AsRef<str>,
+    version: i64,
+    log_buffer_size: usize,
+) -> Result<DeltaTable, DeltaTableError> {
+    let table = DeltaTableBuilder::from_uri(table_uri)
+        .with_version(version)
+        .with_log_buffer_size(log_buffer_size)
+        .load()
+        .await?;
+    Ok(table)
+}
+
```

Comment:
We have been discouraging more use of these top level methods, I would prefer this function not be added and instead users who are interested in buffering to use the table builder method